packages/frontend/src/components/buy/FundingCard.jsx

```diff
@@ -99,23 +99,17 @@ export const FundingCard = ({ title, subTitle, actions, link }) => {
                     <Translate id={subTitle} />
                     {link && (
                         <Link>
-                            <a href={link.url} target='_blank' rel='noreferrer' >
+                            <a href={link.url} target="_blank" rel="noreferrer">
                                 <Translate id={link.title} />
                             </a>
                         </Link>
                     )}
                 </SubTitle>
             </TextWrap>
             <FundingTypeWrap>
-                {actions
-                    .filter(({ disabled }) => !disabled)
```

Comment:
Without the disabled filter, moonPay will be shown as a TopUp method regardless if it is available in a region. 

See https://github.com/mynearwallet/near-wallet/blob/ea5dcd954eaae5634ab67203cf4f0b44ad4bbebb/packages/frontend/src/config/buyNearConfig.js#L23

and [`isMoonPayAvalaible` ](https://github.com/mynearwallet/near-wallet/blob/ea5dcd954eaae5634ab67203cf4f0b44ad4bbebb/packages/frontend/src/utils/moonpay.js#L12)

Reply:
It is made by design - instead of removing MoonPay block from layout we make it look "disabled" - gray and low contrast.
It helps to avoid bumpy layout behaviour during loading time.

Reply:
Good point :)